winterfell/utils/constants.js

```diff
@@ -22,3 +22,12 @@ module.exports.biscuitKeys = {
   idmeClientId: 'id-me-client-id',
   idmeSecretId: 'id-me-client-secret'
 };
+
+module.exports.ERROR_CODES = {
```

Comment:
I would argue that since the error originates from the backend, so it makes sense to have its message stored there too. This allows some nice decoupling of the front and back ends. It would be brittle to expect the client to maintain a hashmap of error codes that must stay in sync with the set of error codes the server may produce.